[2024-05-14T06:20:22.994+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T06:20:23.003+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_jobs104.data_crawler_detail manual__2024-05-14T06:20:19.531102+00:00 [queued]>
[2024-05-14T06:20:23.006+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_jobs104.data_crawler_detail manual__2024-05-14T06:20:19.531102+00:00 [queued]>
[2024-05-14T06:20:23.007+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-14T06:20:23.011+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_crawler_detail> on 2024-05-14 06:20:19.531102+00:00
[2024-05-14T06:20:23.014+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=188) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T06:20:23.014+0000] {standard_task_runner.py:63} INFO - Started process 190 to run task
[2024-05-14T06:20:23.015+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_jobs104', 'data_crawler_detail', 'manual__2024-05-14T06:20:19.531102+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_jobs104.py', '--cfg-path', '/tmp/tmpoi2zu654']
[2024-05-14T06:20:23.016+0000] {standard_task_runner.py:91} INFO - Job 35: Subtask data_crawler_detail
[2024-05-14T06:20:23.033+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_jobs104.data_crawler_detail manual__2024-05-14T06:20:19.531102+00:00 [running]> on host 2c2b6f80804c
[2024-05-14T06:20:23.068+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_jobs104' AIRFLOW_CTX_TASK_ID='data_crawler_detail' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T06:20:19.531102+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-14T06:20:19.531102+00:00'
[2024-05-14T06:20:23.069+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T06:20:23.074+0000] {logging_mixin.py:188} INFO - data_crawler_detail !!!
[2024-05-14T06:20:23.074+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-14T06:20:23.075+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T06:20:23.078+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_jobs104, task_id=data_crawler_detail, run_id=manual__2024-05-14T06:20:19.531102+00:00, execution_date=20240514T062019, start_date=20240514T062023, end_date=20240514T062023
[2024-05-14T06:20:23.115+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T06:20:23.126+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-14T06:20:23.127+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
